apps/consoledot-rhosak/src/utils.ts

```diff
@@ -0,0 +1,26 @@
+import type { KafkaInstanceEnhanced } from "consoledot-api/src/transformers/kafkaRequestToKafkaInstanceEnhanched";
+import { ReadyStatuses, SuspendedStatuses } from "ui-models/src/models/kafka";
+import type { Status } from "ui-models/src/models/kafka";
+
+export function canOpenConnection(instance: KafkaInstanceEnhanced): boolean {
+  return instance ? ReadyStatuses.includes(instance.status) : false;
+}
+
+export const isUserOwnerOrAdmin = (
+  owner: string,
+  username: string,
+  isOrgAdmin: boolean
+) => {
+  return owner === username || isOrgAdmin;
+};
+
+export const canChangeOwner = (
+  owner: string,
+  status: Status,
+  username: string,
+  isOrgAdmin: boolean
+) => {
+  return (
+    owner === username || (isOrgAdmin && !SuspendedStatuses.includes(status))
+  );
+};
```

Comment:
As we now have a "user" type returned by `useUser`, maybe we can think of returning these functions from that hook, so we don't have to pass the username around.